LDY/Day4/로또의 최고순위와 최저순위.js

```diff
@@ -0,0 +1,36 @@
+function solution(lottos, win_nums) {
+  let correctCounts = [0, 0];
```

Comment:
```suggestion
  const correctCounts = [0, 0];
```
배열이나 객체 자체를 새로 대체하지 않고 내부 값만 수정하는 경우라면 const로 선언하는 게 더 좋아요!
const는 변수 재할당만 막고, 내부 값 변경은 가능합니다.